libhb/scan.c

```diff
@@ -1376,6 +1376,12 @@ static void LookForAudio(hb_scan_t *scan, hb_title_t * title, hb_buffer_t * b)
         }
     }
 
+	// Append input bitrate in kbps to the end of the description
+	char in_bitrate_str[12];
+	sprintf(in_bitrate_str, " %d", audio->config.in.bitrate / 1000);
```

Comment:
As @bradleysepos said, this needs "kbps" units.  It also needs to be surrounded with parenthesis like the other descriptive pieces that are added to the full audio description.